samples/multi_api_shell/sensor_shell/app.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Trackunit A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c2 {
+	lis2dh@0b {
+		compatible = "st,lis2dh";
+		reg = <0x0b>;
+		label = "sensor2";
+	};
+
+	lis2dh@0c {
+		compatible = "bosch,bme280";
+		reg = <0x0c>;
+		label = "sensor3";
```

Comment:
labels are deprecated

Reply:
I know, it is just an example showing the use of DT_PROP and DEVICE_DT_PROPERTY with the same node :) This file is just for demonstration purposes